core/src/components/input/input.tsx

```diff
@@ -182,7 +182,8 @@ export class Input implements ComponentInterface {
    * `"stacked"`: The label will appear smaller and above the input regardless even when the input is blurred or has no value.
    * `"fixed"`: The label has the same behavior as `"start"` except it also has a fixed width. Long text will be truncated with ellipses ("...").
    */
-  @Prop() labelPlacement: 'start' | 'end' | 'floating' | 'stacked' | 'fixed' = 'start';
+  @Prop() labelPlacement: 'start' | 'end' | 'floating' | 'stacked' | 'fixed' =
```

Comment:
We should explore/think on this more. Using a call expression and variable reference shows up in the generated `api.txt` and docs output. 

We could probably remove the default value and infer the label placement based on the theme (if it is not set) in the implementation. This would make the generated output clearer.

Reply:
Updated in https://github.com/ionic-team/ionic-framework/pull/29268/commits/8d4e8f5296966eb538b7c598380af306359e9562. I used the pattern from the `alignment` prop in `ion-popover`.